addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -1888,38 +1888,39 @@ $-product-radius-map: (
 
 // Unified product image aspect ratios
 
-%o-wsale-product-img {
+%o-wsale-shop-thumb {
     object-fit: cover;
     object-position: center;
+    aspect-ratio: var(--o-wsale-shop-thumb-aspect-ratio, 1);
 }
 
-@mixin o-wsale-product-img-ratio($ratio) {
-    .o_cart_product_image img,
-    .o_sale_product_configurator_img img,
-    .o_wsale_comparison_bottom_bar img,
-    .o_wsale_compare_table_column img,
-    .product-card img {
-        aspect-ratio: $ratio;
-        @extend %o-wsale-product-img;
-    }
+// Shop ratio classes
+.o_wsale_shop_ratio_16_9 {
+    --o-wsale-shop-thumb-aspect-ratio: 16/9;
+}
+.o_wsale_shop_ratio_4_3  {
+    --o-wsale-shop-thumb-aspect-ratio: 4/3;
+}
+.o_wsale_shop_ratio_6_5  {
+    --o-wsale-shop-thumb-aspect-ratio: 6/5;
+}
+.o_wsale_shop_ratio_1_1  {
+    --o-wsale-shop-thumb-aspect-ratio: 1/1;
```

Comment:
Why we still need this? leftover form the previous implementation?
If the reason is that `o_wsale_shop_ratio_1_1` is still generated via the python getter, then let's not create a class just for that, let's just adapt the getter to return nothing  the forward-port instead